R/calc_stats.R

```diff
@@ -24,7 +24,7 @@ calc_stats <- function(obs, model, depths, folder = ".", NH, flake_nml, out_time
                                nml_file = flake_nml, long = TRUE,
                                out_time = out_time, out_hour = out_hour)
 
-    stats <- sum_stat(fla_long, obs, depth = TRUE, strat = FALSE)
+    stats <- sum_stat(fla_long, obs, depth = TRUE)
```

Comment:
I don't see sum_stat? Is that a file you've added or changed? As long as you're sure this doesn't lead to crashes, fine by me :)

Reply:
sum_stat is from the legendary gotmtools package. Which is 'relatively' stable...

Reply:
changed it to gotmtools::sum_stats 9ba3c13905eb542b4794694b4df276405e5ab681